components/post_view/channel_intro_message/channel_intro_message.jsx

```diff
@@ -34,7 +34,7 @@ export default class ChannelIntroMessage extends React.PureComponent {
         channelProfiles: PropTypes.array.isRequired,
         enableUserCreation: PropTypes.bool,
         isReadOnly: PropTypes.bool,
-        teamIsGroupConstrained: PropTypes.bool.isRequired,
+        teamIsGroupConstrained: PropTypes.bool,
```

Comment:
`team.group_constrained` is sometimes null if not set.  So, either make this as not required or wrapped it with `Boolean(team.group_constrained)`.